pkgs/rk3588-firmware/default.nix

```diff
@@ -0,0 +1,28 @@
+{ pkgs, stdenv, ... }:
+
+stdenv.mkDerivation rec {
+  pname = "rk3588-firmware";
+  version = "2024.09.14";
+
+  src = fetchGit {
+    url = "https://github.com/friendlyarm/sd-fuse_rk3588.git";
```

Comment:
does this need to be your fork? we should probably move it to the org if so

Reply:
This is the official source (it uses the firmware unmodified).  I'm expecting my fork to be replaced by extra code in the dogebox repo.